crates/rbuilder/src/mev_boost/mod.rs

```diff
@@ -31,6 +31,16 @@ const GZIP_CONTENT_ENCODING: &str = "gzip";
 const BUILDER_ID_HEADER: &str = "X-Builder-Id";
 const API_TOKEN_HEADER: &str = "X-Api-Token";
 
+/// We don't have nice error codes for relay errors so we have to parse looking for substrings :(
+/// Simulation error base.
+const SIM_FAILED_SUBSTRING: &str = "simulation failed";
+/// Any error containing SIM_FAILED_SUBSTRING and any of SIM_FAILED_NON_CRITICAL_ERRORS is not critical so we should not stop block building.
+const SIM_FAILED_NON_CRITICAL_ERRORS: &[&str] = &[
+    "blacklisted address", // Generated block is good but contains a blacklisted address. This happens if we don't use an OFAC blacklist but the relay does.
```

Comment:
Nit:

Use kinder language. 

```suggestion
    "blocklisted address", // Generated block is good but contains a blocklisted address. This happens if we don't use an OFAC blocklist but the relay does.
```